scripts/regions.rb

```diff
@@ -7,60 +7,46 @@
 require 'parallel'
 
 data_dir = './_data'
-websites = JSON.parse(File.read("#{data_dir}/all.json"))
-regions = YAML.load_file("#{data_dir}/regions.yml")
-tmp_dir = '/tmp'
-regions.insert(0, { 'id' => 'int', 'name' => 'global' })
-git_dir = Dir.glob('.git')
-FileUtils.cp_r(git_dir, "#{tmp_dir}/") unless File.exist?("#{tmp_dir}/.git")
+categories = JSON.parse(File.read("#{data_dir}/categories.json"))
+regions = YAML.load_file("#{data_dir}/regions.yml").map { |r| [r['id'], r] }.to_h
```

Comment:
```suggestion
regions = YAML.load_file("#{data_dir}/regions.yml").map { |r| [r['id'], { 'name' => r['name'] }] }.to_h
```
For consistency with 'int' key.